config/locales/models/export_templates/en.yml

```diff
@@ -0,0 +1,19 @@
+en:
+  activerecord:
+    models:
+      export_template: Export template
+    attributes:
+      export_template:
+        hints:
+          name: "The name will be visible by you and the other instructors"
+          tiptap_default_dossier_directory: "How would you like to name the directory containing the documents of a folder?"
+          tiptap_pdf_name: "How would you like to name the pdf file containing all the user's answers?"
+        name: Name
+        tiptap_default_dossier_directory: Nom du répertoire
+        tiptap_pdf_name: "Nom de l'export"
```

Comment:
a anglaitisé (quoi, je parle pas bien ?)

Reply:
Copy/paste is hell !